azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+pool:
+  name: VSEng-MicroBuildVS2017
+  demands: msbuild
```

Comment:
Does this have to be a boolean?

Reply:
Not sure I understand your question. This means "the thing this runs on must have the capability named `msbuild`", which is probably a bit redundant since everything in the pool does but it was part of the UI->YAML conversion.